duckduckgo/Android
# Hotfix/5.85.1

Task/Issue URL: https://app.asana.com/0/inbox/1125189844075764/1200285697263489/1200286837564515
Tech Design URL: 
CC: 

**Description**:
This PR updates the bundled version of tds to the latest one

**Steps to test this PR**:
1. Install from the tag `5.81.0`
1. In the table tdsMetadata the etag should be `961c7d692c985496126cad2d64231243`
1. Update from the tag `5.85.0`
1. In the table tdsMedata the etag should be `5c5dda7f1873f3183b141c0739a187ca` which is not correct and references to the old bundled tds data.
1. Update from this branch
1. In the table tdsMetadata the etag should be again `961c7d692c985496126cad2d64231243` 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
